### ISSUE TITLE
Any example

### ISSUE THREAD

**Issue by rachit2jain** [rhdeck/react-native-coreml#2]
Hi,
I have been trying to use the library but I am not able to load the model using compileModel. The app just is stuck there and no response is given. Any idea what the problem is? Also, it will be really helpful if you could provide some example code? Thanks in advance!

**Comment by rhdeck** [comment]
Can you share a sample of what you are trying to do?

**Comment by rhdeck** [comment]
Also, not clear what example you are looking for - something beyond the snippet in the readme?

**Comment by rachit2jain** [comment]
Yes please - a code snippet would be very helpful especially how to load the model.

**Comment by rhdeck** [comment]
I am not sure what you mean by load. Maybe if you were to share a snippet of code giving you trouble I could get a frame of reference to help you from.

**Comment by rachit2jain** [comment]
Load as in compile.

I imported the compileModel from ‘react-native-coreml’

Then I’m trying to compile the model using compileModel command.

import { compileModel } from ‘react-native-coreml’
onClassification = async => {
    const modelPath = await compileModel(‘../models/xyz.mlmodel’)
}
It returns me an error - evaluating RNModel.compileModel: undefined is not an object.

**Comment by matthewlui** [comment]
To library author, you should state that, the path you need to init the model, is a "ABSOLUTE PATH" instead of a relative path that I think normal user will always expect. This guy obviously pass inside a relative path... @rhdeck  @rachit2jain find an absolute path of that model and pass inside, work like a charm

**Comment by rhdeck** [comment]
Thank you l! That is helpful.

**Comment by rhdeck** [comment]
I would love to know what path was being passed - if there is a common
reference point the lib  might be able to normalize it.

**Comment by rhdeck** [comment]
Thank you those are both good advice. Can I ask you to file them as issues
so commits can map to them?

**Comment by rachit2jain** [comment]
Thank you so much guys for your help. I have successfully compiled the model. 
I am trying to use classifyImage function but I am not getting anything back from the library.
So I tried looking into the library and found an EXC_BREAKPOINT error in line 
`let handler = VNImageRequestHandler(ciImage: image!)
`So I checked the source URL of the image file and it is right. Any ideas why it is giving me this error?

**Comment by davidpfahler** [comment]
An example app would really help. The instruction in the README leave a lot of room for mistakes, many of which I am apparently making:

I am getting the following error as displayed in the iPhone simulator: `TypeError: null is not an object (evaluating 'RNCoreML.mainBundlePath')` and the first item in the stack trace point to line [14 in index.js](https://github.com/rhdeck/react-native-coreml/blob/master/index.js#L14) of this project:

`const { mainBundlePath, mainBundleURL } = RNCoreML;`

Also, I would very much appreciate it if you could provide an example of loading the `.mlmodel` file from the bundle instead of downloading it from the web.

Thank you very much in advance.

**Comment by chase-robbins** [comment]
> To library author, you should state that, the path you need to init the model, is a "ABSOLUTE PATH" instead of a relative path that I think normal user will always expect. This guy obviously pass inside a relative path... @rhdeck @rachit2jain find an absolute path of that model and pass inside, work like a charm

This should be added to the readme.
